sdk/core/core-lro/src/lroPoller/bodyPolling.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { failureStates, LROStatus, RawResponse, successStates } from "./models";
+
+function getProvisioningState(rawResponse: RawResponse): string {
+  const { properties, provisioningState } = rawResponse.body ?? {};
+  const state: string | undefined = properties?.provisioningState ?? provisioningState;
+  return state?.toLowerCase() ?? "succeeded";
```

Comment:
we can assume success?

Reply:
Yeah this is how the autorest test server behaves.